nifi-commons/nifi-sensitive-property-provider/pom.xml

```diff
@@ -23,7 +23,19 @@
     <artifactId>nifi-sensitive-property-provider</artifactId>
     <properties>
         <aws.sdk.version>2.17.1</aws.sdk.version>
+        <gcp.sdk.version>20.6.0</gcp.sdk.version>
     </properties>
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
```

Comment:
What's the benefit of declaring this dependency, rather than just the library dependency on lines 79-82?

Reply:
It's the recommended practice for using the Google Cloud Java SDK